hack/update-codegen.sh

```diff
@@ -22,7 +22,7 @@ function echoerr {
 }
 
 ANTREA_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../" && pwd )"
-IMAGE_NAME="antrea/codegen:kubernetes-1.29.2-build.2"
+IMAGE_NAME="antrea/codegen:kubernetes-1.29.2-build.3"
```

Comment:
One question: I assume this image is already uploaded?

Reply:
Yes, I already pushed it